e2e/utils/fixture.ts

```diff
@@ -91,6 +93,25 @@ export class Fixture {
     return fixture;
   }
 
+  private async setNpmRegistry(packageManager: PackageManager): Promise<void> {
+    if (packageManager === "pnpm") {
+      await this.exec(`echo "registry=${REGISTRY}" > .npmrc`);
+    }
+  }
+
+  private async initializeNpmEnvironment(packageManager: PackageManager): Promise<void> {
```

Comment:
The methods already have access to `this.packageManager`

Reply:
Maybe `initializePackageManagerSettings` could be a more generic name for this method, and then the logic of the currently separate `setNpmRegistry` could be folded into it? Or do they need to be separate steps?

Reply:
I already tried this and had to split them out. setNpmRegistry has to happen first, since the rest of the initialization has to happen after running `lerna init`. The registry must be set first for the `npx` command that runs `lerna init`.